Resources/translations/messages.da.yaml

```diff
@@ -312,6 +312,7 @@ menu:
         process_type: Sagstyper
         quick_link: Genveje
         reminder: Påmindelser
+        revenue: Provenue
```

Comment:
It's spelled `Provenu` in Danish (cf. https://sproget.dk/lookup?SearchableText=provenu).

Reply:
Ups.

Reply:
Fixed in: https://github.com/aakb/kontrolgruppen-core-bundle/pull/34/commits/70dcb1fe6ccc2cada3c44731ad56aed46ce80de0